docs/sdks/python/GETTING_STARTED.md

```diff
@@ -43,7 +43,7 @@ client = Client()
 
 users = Users(client)
 
-result = users.create(ID.unique(), 'email@example.com', 'password')
+result = users.create(ID.unique(), email = 'email1@example.com', phone = '+123456789', password = 'password', name = 'user1')
```

Comment:
Why don't we use something like `Walter O'Brien` that's actually a name instead of user1

Reply:
Yeah, that's good idea. Can use `Walter O'Brien` as name instead of `user1`, look more fancy and real.
Will make the changes and do a commit again.

Reply:
Hi @gewenyu99, committed the changes. Checkout once
Thank you.